cmd/clusterctl/client/init.go

```diff
@@ -197,11 +204,16 @@ func (c *clusterctlClient) InitImages(options InitOptions) ([]string, error) {
 func (c *clusterctlClient) setupInstaller(cluster cluster.Client, options InitOptions) (cluster.ProviderInstaller, error) {
 	installer := cluster.ProviderInstaller()
 
+	var noKindMatchErr *meta.NoKindMatchError
 	providerList, err := cluster.ProviderInventory().List()
-	if err != nil {
+	if !errors.As(err, &noKindMatchErr) {
```

Comment:
`errors.As` is ment to be used to retrieve an underlying error. Have you considered using `errors.Is` instead? `errors.Is` does check for the target erorr in the chain of wrapped errors so I believe it will work in this case.

Reply:
Yes, `errors.Is` is working now don't know why it was not working but now fixed 😊